### DISCUSSION TITLE
where is config.toml?

### DISCUSSION THREAD

**Discussion by torstenwerner** [lima-vm/lima#411] [Q&A] [answered]
% lima ls /etc/containerd
    ls: cannot access '/etc/containerd': No such file or directory

How do I configure containerd? My actual question is: how do I configure insecure registries?

**Comment by AkihiroSuda** [comment, marked as answer]
The config file path is `/home/$USER.linux/.config/containerd/config.toml`( in the VM ), but the config file has no effect to registry configuration of nerdctl.
Please use `lima nerdctl --insecure-registry`
https://github.com/containerd/nerdctl/blob/master/docs/registry.md
